modules/cnf-configure_for_irq_dynamic_load_balancing.adoc

```diff
@@ -44,7 +44,7 @@ metadata:
 spec:
   containers:
   - name: dynamic-irq-pod
-    image: "quay.io/openshift-kni/cnf-tests:4.9"
+    image: "quay.io/openshift-kni/cnf-tests:4.10"
```

Comment:
Please avoid using quay. Use the proper RH image pull spec registry.redhat.io/openshift4/cnf-tests-rhel8:v4.10